content/en/docs/tasks/configure-pod-container/security-context.md

```diff
@@ -374,7 +386,10 @@ See [capability.h](https://github.com/torvalds/linux/blob/master/include/uapi/li
 for definitions of the capability constants.
 
 {{< note >}}
-Linux capability constants have the form `CAP_XXX`. But when you list capabilities in your Container manifest, you must omit the `CAP_` portion of the constant. For example, to add `CAP_SYS_TIME`, include `SYS_TIME` in your list of capabilities.
+Linux capability constants have the form `CAP_XXX`.
+But when you list capabilities in your Container manifest, you must
```

Comment:
nit: change Container to lowercase (container)?

Reply:
I am a little conservative on this. I saw some changes rewriting 'Container' as 'container' even when we are referring to the Container resource (just like a Pod resource), but I don't think those changes are correct.
In this specific case, maybe 'Container' is more appropriate than 'container' because we are talking about the manifest.
Changing this to 'container' is okay though.

Reply:
Sure, my comment is more of a note or question.